root/static/scripts/edit/externalLinks.js

```diff
@@ -212,7 +212,8 @@ export class ExternalLinksEditor
             } else if (!isValidURL(link.url)) {
               error = l('Enter a valid url e.g. "http://google.com/"');
             } else if (isShortened(link.url)) {
-              error = l("Please don't use shortened URLs.");
+              error = l(`Please don’t enter bundled/shortened URL,
```

Comment:
Small grammar fix: URL should be plural here.
```suggestion
              error = l(`Please don’t enter bundled/shortened URLs,
```